### ISSUE TITLE
Forum: compiling across the stack

### ISSUE THREAD

**Issue by benmiller314** [benmiller314/cdm2021spring#11]
Periodically in the semester, I will ask you to develop proposals for a final project that will allow you to integrate – and extend – what you’ve learned about composing digital media. This is the third of those proposals; you may wish to review [the first](4) and [second](8). There is no minimum length.

Consider:
* What further possibilities of web, graphic, or sound design might you want to explore in connection with other people, and/or other media?
* What projects have you started that don't yet feel finished, that you might want to spend another couple of weeks on?
* What tools or genres were you hoping to try in this course that haven't been assigned?

NB: This final project can be a solo or group project, or, with enough shared enthusiasm, a collective project that we divide up among all members of the class. But bear in mind [we only have a few weeks](https://benmiller314.github.io/cdm2021spring/schedule#unit-5) after the web unit ends!

As always on the forum, replying directly to each other is optional but encouraged: just use the @ symbol to flag someone by username.

**Comment by reaial** [comment]
I feel like my website will be the one thing that I want to work on a little more, I do believe that my other projects are really where I want them and they will not benefit from any more work it might actually mess them up. 

In terms of any other form of media, I do think that we played with what we would have for each medium as much as this course allows. I do think that there are other things like music, or more visual things that we could've gotten into but that was not really apart of the plan. 

So in conclusion I will most likely want to work on my website for the final project, to make sure my grade will reflect its work, but if someone has a group collaboration that I find interesting enough I might switch.

**Comment by patrickjmeyer** [comment]
Late to the game, but I love the concept of animation that Abby, Garrett, Greg, and others have touched on so far in the previous issues. I feel like it would allow me to both practice skills that I've learned in this class and also flex the specialties I already had coming in. I'd need a group to do this, though, because I'm no good with anything visual!
